spec/helpers/visit_helper_spec.rb

```diff
@@ -84,7 +84,9 @@ def visit
     end
 
     it "provides a formatted date for when a response may be sent out" do
-      expect(helper.when_to_expect_reply(Date.parse("2014-10-03"))).to eq("Monday  6 October")
+      Timecop.travel(Date.parse("2014-10-03")) do
+        expect(helper.when_to_expect_reply).to eq("Friday  3 October")
```

Comment:
Might be worth seeing if we can kill this probably-pointless space? @matharden: any idea why there'd be a double-space in the output here?